metadata-ingestion/src/datahub/check/docker.py

```diff
@@ -3,28 +3,30 @@
 
 import docker
 
+ENSURE_EXIT_SUCCESS = [
+    "kafka-setup",
+    "elasticsearch-setup",
+]
+
 REQUIRED_CONTAINERS = [
     "elasticsearch-setup",
     "elasticsearch",
     "datahub-gms",
     "datahub-mce-consumer",
     "datahub-frontend-react",
     "datahub-mae-consumer",
-    "kafka-topics-ui",
     "kafka-rest-proxy",
     "kafka-setup",
-    "schema-registry-ui",
     "schema-registry",
     "broker",
     "kibana",
```

Comment:
Prob also don't need kibana and kafka-rest-proxy

Reply:
fixed!